docs/utilities.md

```diff
@@ -800,25 +800,32 @@ However, this can be easily resolved by using more hex values from the output by
 
 ##### $KDBCONFIG/process.csv
 
-The process file should contain the RDB instances by specifying a different **`port`** and **`procname`** column. Set the **`load`** column to **`${KDBCODE}/processes/rdb.q`**.
+The process file should contain the RDB instances by specifying a different **`port`** and **`procname`** column. Set the **`load`** column to its respective **`${KDBCODE}/processes/rdb{i}.q`** file.
```

Comment:
What's different in each of the `rdb{i}.q` files? I thought we just had separate rdbsub.csv files for each process?

Reply:
Oh yes thats right. I have made the changes.